connect/eaas/core/proto.py

```diff
@@ -276,3 +277,20 @@ def deserialize(cls, raw):
             )
 
         return cls(version=version, message_type=message_type)
+
+
+@dataclasses.dataclass
```

Comment:
These dataclasses are not part of the websocket protocol. If we decide to maintain these classes in the core lib, we should put them in another module.